recipes/geocat-viz/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set name = "geocat-viz" %}
+{% set version = "2021.08.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NCAR/{{ name }}/archive/v{{ version }}.tar.gz   # For building from a specific tag's tarball
+  sha256: 6624ffb855cbe717a8eb624c7fa313a47bf1b3827e22f28f4dec8b2e298d47cd
+  # git_tag: v{{ version }}   # For building from a specific tag
+  # git_rev: main # For building from main branch for debugging purposes
+  # git_url: https://github.com/NCAR/{{ name }}.git
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed -vv .
+
+requirements:
+  build:
+    - python                              # [build_platform != target_platform]
+    - pip                                 # [build_platform != target_platform]
+    - setuptools                          # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+  host:
+    - python >=3.8
+    - pip
+    - setuptools
+  run:
+    - python >=3.8
+    - cartopy
+    - cmaps
+    - numpy
+    - scikit-learn
+    - xarray
+
+test:
+  requires:
+    - pip
+    - cmaps
+    - numpy
+    - matplotlib
+    - scikit-learn
```

Since most of these are in `run`, you don't need them. Also, unless you need a gui, better to use `matplotlib-base`, though I'm confused why it's only needed for tests?
```suggestion
    - matplotlib-base
```

---

I honestly don't know, I'm haven't developed in geocat-viz, but since I handled our team's other conda-forge recipes, I was chosen to get this package onto conda-forge. if it builds with those changes, I'm happy to make them.

---

matplotlib is in  `requirements.txt` in the project's source repo, so it should also be deleted here and added above.